manifests/server.pp

```diff
@@ -161,6 +161,7 @@
     conn_keep_alive     => $read_conn_keep_alive,
     conn_lifetime       => $read_conn_lifetime,
     confdir             => $confdir,
+    require             => Package[$puppetdb_package],
```

Comment:
I'd say so, I think we should produce a graph and manually double check this maybe. I can see a few just in code, wouldn't hurt to visually check it.